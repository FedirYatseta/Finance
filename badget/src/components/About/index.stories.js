import React from 'react';

import About from '.';

export default {
    title: 'Example/About',
    component: About,
};

const Template = (args) => <About {...args} />;

export const Item = Template.bind({});
Item.args = {


};