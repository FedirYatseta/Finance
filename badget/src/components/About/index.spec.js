

//import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import About from '.';



describe('About', () => {
  it('should show About', () => {
    const component = renderer.create(
      <About />,
    );
    expect(component).toMatchSnapshot();

  })
});