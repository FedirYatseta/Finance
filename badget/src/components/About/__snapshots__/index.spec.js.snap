// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should show About 1`] = `
<h1>
  About
</h1>
`;

exports[`App should show App 1`] = `
<h1>
  About
</h1>
`;
